src/components/Homepage/components/navigation.tsx

```diff
@@ -57,7 +63,12 @@ const Navigation: React.FC<NavigationProps> = ({ hasBg = false }) => {
           <AuthButtons />
         </div>
       </div>
-      <div className="block lg:hidden">
+      <div className="flex items-center gap-3 lg:hidden">
+        {user && (
+          <Avatar className="h-9 w-9">
+            <AvatarFallback>{`${first_name[0]}${last_name[0]}`}</AvatarFallback>
```

Comment:
In line with the comment above
```suggestion
            <AvatarFallback>{`${user.first_name[0]}${user.last_name[0]}`}</AvatarFallback>
```

Reply:
Okay. Thank you.. This has been catered for